src/ripple/rpc/impl/Handler.cpp

```diff
@@ -147,7 +147,7 @@ HandlerTable HANDLERS({
     {   "unl_score",            byRef (&doUnlScore),            Role::ADMIN,   NO_CONDITION     },
     {   "validation_create",    byRef (&doValidationCreate),    Role::ADMIN,   NO_CONDITION     },
     {   "validation_seed",      byRef (&doValidationSeed),      Role::ADMIN,   NO_CONDITION     },
-    {   "wallet_accounts",      byRef (&doWalletAccounts),      Role::USER,  NEEDS_CURRENT_LEDGER  },
```

Comment:
Fantastic!  Suddenly, over a dozen things work before we're synched that didn't work before...